src/formatters.ts

```diff
@@ -7,8 +7,8 @@ export const displayIdentity = (address: string, chainId?: number) => {
   const addr = `${address.slice(0, 6)}...${address.slice(address.length - 4)}`
   switch (chainId) {
     case 1: return `did:eth:${addr}`
-    case 30: return `did:rsk:${addr}`
-    case 31: return `did:rsk:testnet:${addr}`
+    case 30: return `did:ethr:rsk:${addr}`
+    case 31: return `did:ethr:rsk:testnet:${addr}`
```

Comment:
Why don't using `createDidFormat`?

Reply:
hahahah! we are!!! In the next PR you are looking at, this function is going away completely!! I didn't remove it in this PR since I wasn't touching the code that interacted with it.

Reply:
🙌